docs/application/native/get-started/overview.md

```diff
@@ -1,22 +1,140 @@
 # Get Started with Native Applications
 
-When creating your first Tizen native application, you can select from the following options, which all use the C language:
+<style>
+#main:before, #main:after {
+    content: "";
+    display: table;
+}
+.docs-ui-started [class^="docs-ui-"] {
```

Comment:
Request to the designing team:
The CSS alignment can be modified in a better way.